modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -57,14 +58,36 @@ $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused" :true } }'
 ----
 
-. To configure the OVN-Kubernetes cluster network provider, enter the following command:
+. To configure the OVN-Kubernetes cluster network provider with one of the following commands:
+
+** To specify the network provider without changing the cluster network IP address block, enter the following command:
 +
 [source,terminal]
 ----
 $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
 ----
 
+.. Optional: To specify a different cluster network IP address block, enter the following command:
```

Comment:
```suggestion
** To specify a different cluster network IP address block, enter the following command:
```

Since you need to use one or the other of these commands, I think it's clearer to present them as parallel options in this way.